lms/static/scripts/frontend_apps/components/test/FilePickerApp-test.js

```diff
@@ -135,6 +142,80 @@ describe('FilePickerApp', () => {
     });
   });
 
+  context('when deeplinkingAPI is present', () => {
+    const deeplinkingAPIPath = '/lti/1.3/deeplinking/formfields';
+    const deeplinkingAPIData = { some: 'data' };
+    let fakeAPICall;
+    let fakeFormFields;
+
+    beforeEach(() => {
+      fakeConfig.filePicker.deeplinkingAPI = {
+        path: deeplinkingAPIPath,
+        data: deeplinkingAPIData,
+      };
+
+      fakeAPICall = sinon.stub();
+      fakeFormFields = { JWT: 'JWT VALUE' };
+
+      fakeAPICall
+        .withArgs(sinon.match({ path: deeplinkingAPIPath }))
+        .resolves(fakeFormFields);
+
+      $imports.$mock({
+        '../utils/api': { apiCall: fakeAPICall },
+      });
+    });
+
+    it('calls deeplinkingAPI when content is selected', async () => {
```

Comment:
```suggestion
    it('fetches form field values via deeplinking API when content is selected', async () => {
```

This test covers both calling the API and populating the form with the response, so I think it is helpful for the description to mention that.